src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+
+import duke.commands.CommandHandler;
+import duke.exception.IllegalTaskCommandException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+
+    /**
+     * Starts the Task Manager program.
```

Comment:
Can remove the first line here. The main job is to run the task manager loop. So `Starts ...` may be misleading